docs/core/logging.md

```diff
@@ -640,6 +640,9 @@ To append additional keys in your logs, you can use the `StructuredArguments` cl
             }
         ```
 
+???+ warning "Do not use arguments with reserved keys"
```

Comment:
I would consider just adding a notice/warning directly in [this section](https://docs.powertools.aws.dev/lambda/java/core/logging/#standard-structured-keys) so that customers don't need to jump around, similar to what we did in TypeScript [see wording](https://docs.powertools.aws.dev/lambda/typescript/latest/core/logger/#standard-structured-keys).

However I also noticed that the table for structured keys in the docs might need to be updated. In the code you correctly set as "reserved" keys some of them, but the table mixes other keys from the context that are not standard keys (i.e. `functionName`, `functionVersion`, etc.). Perhaps better split that table and add these under the section where you talk about context instead.

Reply:
I will work on this change and send a new commit.

Regarding the table, I believe this is already fixed in the preview documentation, did you look at the `v1` docs? Here is a link to the preview docs tables. There it is split into two tables: https://docs.powertools.aws.dev/lambda/java/preview/core/logging/#standard-structured-keys

Is this what you mean?

Reply:
I updated the docs and deployed a preview here: https://dealn7fl31ram.cloudfront.net/core/logging/#standard-structured-keys.

I added the same note like in TypeScript right at the beginning where the documentation introduces structured keys.

I kept the warnings in the respective section of `StructuredArguments` and `MDC` with the details on how each method is treated.